model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -83,6 +83,10 @@ class LicenseInfoResolver(
                 licenseInfo.concludedLicenseInfo.concludedLicense?.also {
                     originalExpressions += ResolvedOriginalExpression(expression = it, source = LicenseSource.CONCLUDED)
                 }
+
+                licenseInfo.declaredLicenseInfo.authors.takeIf { it.isNotEmpty() && addAuthorsToCopyrights }?.also {
+                    locations += resolveCopyrightFromAuthors(it)
```

Comment:
TBH, I still don't understand the details *why* this is necessary. Can it be that the real reason is not related to *having* a concluded license, but to *not* having a declared license, and so the below logic does not kick in?

Reply:
Or is it more like:

1. Copyrights / authors are associated with specific licenses.
2. A concluded license trumps all other licenses.
3. If both a concluded license and a declared license are present, but only the declared license has a Copyright / author associated, that will not be used as the concluded license it taken instead of the declared license.

Reply:
As @mnonnenmacher just explained to me, it's probably a mixture of both of the above: The problems occurs if there is a concluded license that is not also present as a declared license (which is esp. the case if there are no declared licenses).

Reply:
Yes, I guess it happens if a.) _declared_ and _concluded_ license are not the same (overwritten in curation) or there is no _declared_ license at all.

Reply:
Hmm, you "guess" what you're fixing here? 😉 Us all having some more confidence in understanding what exact is being fixed would be great.

To document that, I propose a more concise commit message like the following (which also drops the explicit mention of the disclosure document, as the fix is not specific to that, and the fix is in the model, not in the reporter):

```
fix(model): Apply author to copyright mapping also for concluded licenses

As concluded licenses are supposed to trump other licenses, they are
usually taken instead of declared and detected licenses in reports. When
doing do, and the concluded license is not among the other licenses, any
author information that is configured to be mapped to copyright would be
dropped. To prevent that, apply the same author to copyright mapping that
is done for declared licenses also for concluded licenses.
```

Reply:
Thanks, I took over the commit message.